spec/views/shared/_help-tooltip.html.erb_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe 'shared/_help-tooltip.html.erb' do
+  it 'renders with block as USWDS tooltip' do
+    tooltip_text = 'foobar'
+    render('shared/help-tooltip') { tooltip_text }
+
+    expect(rendered).to have_css(".help-tooltip__button[title='#{tooltip_text}']")
```

Comment:
Then this might be something like

```suggestion
    expect(rendered).to have_css(".help-tooltip__button[title='\"foo\" \"bar\"']")
```

Reply:
> Then this might be something like

Updated in b17c579. (Didn't need to change this assertion)